libfwupdplugin/fu-self-test.c

```diff
@@ -2908,6 +2910,143 @@ fu_device_retry_hardware_func(void)
 	g_assert_cmpint(helper.cnt_failed, ==, 2);
 }
 
+static void
+fu_bios_attrs_load_func(void)
+{
+	gboolean ret;
+	g_autofree gchar *test_dir = NULL;
+	g_autoptr(FuContext) ctx = fu_context_new();
+	g_autoptr(GError) error = NULL;
+	g_autoptr(FuBiosAttrs) p620_attrs = NULL;
+	g_autoptr(GPtrArray) p620_items = NULL;
+	g_autoptr(FuBiosAttrs) p14s_attrs = NULL;
+	g_autoptr(GPtrArray) p14s_items = NULL;
+	g_autoptr(FuBiosAttrs) xps9310_attrs = NULL;
+	g_autoptr(GPtrArray) xps9310_items = NULL;
```

Comment:
maybe some sorting here, and move the g_autoptr's to last

Reply:
sorted